genesis/engine/solvers/fem_solver.py

```diff
@@ -1034,6 +1073,18 @@ def get_state_render(self, f):
 
         return vertices, indices
 
+    def get_forces(self):
+        """
+        Get forces on all vertices.
+
+        Returns:
+            torch.Tensor : shape (B, n_vertices, 3) where B is batch size
+        """
+        if not self.is_active():
+            return None
```

Comment:
Not sure if returning none is a good approach. But let's keep it like this for now.

Reply:
I think it's okay for now. Or what do you think would be a better handling? Raise an exception?